docs/source/advanced/loops.rst

```diff
@@ -0,0 +1,399 @@
+.. _loop_customization:
+
+Loop Customization
+==================
+
+Loop customization is an experimental feature introduced in Lightning 1.5 that enables advanced users to write new training logic or modify existing behavior in Lightning's training-, evaluation-, or prediction loops.
+By advanced users we mean users that are familiar with the major components under the Trainer and how they interact with the LightningModule.
+
+In this advanced user guide we will learn about
+
+- how the Trainer runs a loop,
+- the Loop base class,
+- the default loop implementations and subloops Lightning has,
```

Comment:
shouldnt each bullet point start with a capital and end with a period?

Reply:
it's part of one large sentence. 
individually, these wouldn't be sentences

Reply:
Agree with @carmocca this doesn't work well imo. Could we make these full sentences and bullet point them?